datafusion/core/tests/parquet/mod.rs

```diff
@@ -824,9 +863,17 @@ fn create_data_batch(scenario: Scenario) -> Vec<RecordBatch> {
         }
         Scenario::WithNullValues => {
```

Comment:
Could you please also add the "some null/some not null" null values case to this scenario as well? 
```
                make_int_batches_with_null(5, 1, 6),
```